ime/app/src/main/java/com/anysoftkeyboard/keyboards/views/KeyboardViewContainerView.java

```diff
@@ -27,6 +29,7 @@
     private OnKeyboardActionListener mKeyboardActionListener;
     private KeyboardTheme mKeyboardTheme;
     private OverlayData mOverlayData = new OverlayData();
+    private LinearLayout mInlineAutofillView;
```

Comment:
I was hoping that you could re-use the `CandidateView` for this, instead of introducing complex layout (which will cost CPU cycles for the user)

Reply:
I can attempt to use the CandidateView, but I do have some questions if you would be willing to answer them.

1. Is it possible for me to pass already inflated views to the CandidateView?
2. What is the desired functionality if we have both candidates to show and autofill suggestions?

Reply:
1. No. That functionality needs to be added.
2. yeah... I'm not sure, maybe for the MVP your implementation is fine.

I was also wondering if it makes sense that we have _two_ suggestions rows, can they exist together?

Reply:
After goofing around with the CandidateView, I noticed that sometimes the CandidateView is not shown, like in the case of a password field. Does it maybe make sense as a MVP of having 2 suggestion rows?

Reply:
Yes, that's fine